### ISSUE TITLE
Widget doesn't work with boolean values.

### ISSUE THREAD

**Issue by h4nz11** [jmrivas86/django-json-widget#90]
* django-json-widget version:
* Django version: 5.0.2
* Python version: 3.10
* Operating System: Ubtuun

### Description
The widget doesn't display data if the JSONField contains a boolean value. It throws a javascript exception:

```
Uncaught ReferenceError: True is not defined
    at change/:982:186
    at change/:984:7
```
Loaded data in HTML:
```
var json = {'label': 'Metered consumption - house', 'category': 'consumption', 'convertToNegative': True};
```

**Comment by paulocoutinhox** [comment]
Hi,

The value "true" in JSON need be lower case "true" and not "True".

Thanks.

**Comment by h4nz11** [comment]
> Hi,
> 
> The value "true" in JSON need be lower case "true" and not "True".
> 
> Thanks.

Yeah, I've tried to save it with `true` but widget convert it back to "Python" syntax `True`.
